pkg/artifactory/resource_artifactory_virtual_repository_test.go

```diff
@@ -250,6 +250,138 @@ func TestAccVirtualMavenRepository_basic(t *testing.T) {
 	})
 }
 
+func TestAccVirtualRpmRepository(t *testing.T) {
```

Comment:
There is a ticket about support for GPG keys somewhere - and I thought I hadn't implemented them. Maybe I did. Go find that ticket and see if you can close it. It's pretty recent

Reply:
This one? https://github.com/jfrog/terraform-provider-artifactory/issues/285

Reply:
So, GPG is supported? I remember there is no way to clientside validate it There needs to be a test just for GPG support for #285 - if memory serves correct, 2 reasons I couldn't support it were: 
1. no client side validation and
2. Couldn't get RT to accept all the test keys I tried.

Reply:
The tests are using GPG type and they pass, so RT is accepting them. I can add GPG tests to `artifactory_keypair`